pipeline-scripts/release.groovy

```diff
@@ -706,6 +711,19 @@ def openCincinnatiPRs(releaseName, advisory, candidate_only = false,ghorg = 'ope
                                 pr_messages << "This should provide adequate soak time for fast channel PR ${prURLs.fast}"
                             }
 
+                            break
+                        case 'eus':
+                            // currently put eus change in a seperate PR as same as stable channel
+                            labelArgs = "-l 'do-not-merge/hold'"
+                            pr_messages << "Please merge within 48 hours of ${internal_errata_url} shipping live OR a Cincinnati-first release."
+
+                            if (prURLs.containsKey('prerelease')) {
+                                pr_messages << "This should provide adequate soak time for prerelease channel PR ${prURLs.prerelease}"
+                            }
```

Comment:
prerelease channel was only for 4.1 ... this can be omitted as it will never be relevant.